website/docs/r/nsxt_edgegateway_dhcp_forwarding.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_dhcp_forwarding"
+sidebar_current: "docs-vcd-resource-nsxt-edge-dhcp-forwarding"
+description: |-
+  Provides a resource to manage NSX-T Edge Gateway DHCP forwarding configuration.
+---
+
+# vcd\_nsxt\_edgegateway\_dhcp\_forwarding
+
+Supported in provider *v3.10+* and VCD 10.3.1+ with NSX-T.
+
+Provides a resource to manage NSX-T Edge Gateway DHCP forwarding configuration.
+
+## Example Usage
+
+```hcl
+data "vcd_org_vdc" "v1" {
+  org  = "datacloud"
+  name = "nsxt-vdc-datacloud"
+}
+
+data "vcd_nsxt_edgegateway" "testing-in-vdc" {
+  org      = "datacloud"
+  owner_id = data.vcd_org_vdc.v1.id
+
+  name = "nsxt-gw-datacloud"
+}
+
+resource "vcd_nsxt_edgegateway_dhcp_forwarding" "testing-in-vdc" {
+  org             = "datacloud"
+  edge_gateway_id = data.vcd_nsxt_edgegateway.testing-in-vdc.id
+
+  enabled = "true"
+
+  dhcp_servers = [
+    "192.168.0.13",
+    "fe80::aaaa",
+  ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Required) Org in which the NSX-T Edge Gateway is located
```

Comment:
This argument should be optional. It is required if the Provider does not set such field

Reply:
Updated.